### ISSUE TITLE
Support paket pack

### ISSUE THREAD

**Issue by haf** [Albacore/albacore#174]
To replace nuget pack.

Work begun: https://github.com/Albacore/albacore/tree/feature/paket-pack

**Comment by wallymathieu** [comment]
Where the script assumes that paket.bootstrapper.exe is in .paket folder in the same directory as rakefile?

**Comment by haf** [comment]
Currently assumes it's in [`tools/paket.bootstrapper.exe`](https://github.com/Albacore/albacore/blob/master/lib/albacore/cli.rb#L141) - dotfiles and -folders are annoying during cross-platform dev, because they aren't shown with the default `ls` commands.

**Comment by wallymathieu** [comment]
If you interact with paket using rake then it doesn't need to be annoying. Also, it could be good to follow how the paket documentation tells you how to set it up.

**Comment by haf** [comment]
Yes, the CLI doesn't download the full paket, for example, and taken together with your flag it could work really nicely.

**Comment by wallymathieu** [comment]
I've added a gem: ```https://github.com/wallymathieu/paketbootstrapper-gem```
based on: ```https://github.com/ferventcoder/nuget-gem```
that initially only contains the bootstrapper in order to simplify things for me.

**Comment by haf** [comment]
That's a good way of doing it. Should you add the binary as well, to the gem?

**Comment by wallymathieu** [comment]
The only binary I've added to the gem is the bootstrapper, since I imagine the use case being to run paket.bootstrapper and then paket in your scripts.

For nuget exe I've used: 

    spec = Gem::Specification.find_by_name("nuget")
    command = File.join(spec.gem_dir, "bin", "nuget.exe")

in order to get the location of the binary (but since nuget-gem is updated to work on both windows and *nix you could just use ```sh "nuget ..."``` or similar)

**Comment by wallymathieu** [comment]
I've fixed so that the paket.exe is downloaded if it's missing:
https://github.com/wallymathieu/paketbootstrapper-gem/blob/master/_template/bin/paket#L5

**Comment by haf** [comment]
Done.
